addon/mocks/query.js

```diff
@@ -7,7 +7,9 @@ import { partial } from '../utils';
 
 export const composeMockQuery =
   (getTypeForField, getFieldNameAndAlias, createFieldInfo, resolveFieldInfo, getIsInterface, logError) =>
-    (db, options, _, vars, __, { fieldNodes, fragments, returnType, schema }) => {
+    (db, options, _, vars, __, info) => {
```

Comment:
Not a required change: Is the `vars` argument used now? If not, maybe we should change this function signature so it has even more underscores?

Also, could you tell me what `info.variableValues` looks like? I haven't seen this before and I'm just curious.

Reply:
`variableValues` is the variables as they come over the wire in the request body

Reply:
And yes, I think switching to underscores is a good call

Reply:
> maybe we should change this function signature so it has even more underscores

Done 👍